src/content/self-hosted/install/releases.mdx

```diff
@@ -5,8 +5,43 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+
+## 1.4.0
+<TODO: insert release date here once it's published>
+
+### Features
```

Comment:
Lots of features, that's very cool! Can we sort them so that related features are together?

Reply:
Reordered!